.github/workflows/build.yml

```
@@ -0,0 +1,73 @@
+name: Wheels
+
+on:
+  push:
+  workflow_dispatch:
+  release:
+    types:
+      - published
+env:
+  CIBW_TEST_EXTRAS: test
+  CIBW_TEST_COMMAND: pytest {project}/tests
+  CIBW_BUILD_VERBOSITY: 1
+  # Optional: support Universal2 for Apple Silicon with these two lines:
+  CIBW_ARCHS_MACOS: auto universal2
+  CIBW_TEST_SKIP: "*universal2:arm64"
+
+jobs:
+  make_sdist:
+    name: Make SDist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
```

We're not using submodules, but you've probably added this to be safe?
I think `actions/checker@v2` is more recent.
```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: true
```

---

The v2 fails with scm, so for the moment I would keep v1

---

yep, just routinely added them. But removed

---

Weird, it does not look like you're using special `actions/checkout` features.
When you are finished, can you try moving back to v2?

---

It's the pruning of the history I think. scm uses this for the version. So basically waiting for this compatibility

---

Do you mean `setuptools_scm`?
Maybe it needs `fetch-depth: 0` (for v2)?

Please add a FIXME/TODO (+ link to an issue), so we know what/where to look to see if/when the problem is resolved.

---

Yes, `setuptools_scm`

Yes, the fetch-depth may helps I've heard, but not convinced. We can try! Worst case we scrap the version